### PR TITLE
Add section to conditionals about key attribute

### PR DESCRIPTION
**Pull request by chrisvfritz**
@LinusBorg Do you think this is a step in the right direction towards resolving #587? Also, is it short enough and explained in simple enough terms so as not to confuse beginners?

### CODE REVIEW

File: src/v2/guide/conditional.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ It is also possible to add an "else" block with `v-else`:
 <h1 v-else>No</h1>
 ```
 
-### Template v-if
+### Conditional Groups with `v-if` on `<template>`
```
Comment (yyx990803) [inline src/v2/guide/conditional.md]: Does this look okay in the sidebar?

**Comment by yyx990803** [comment]
I like it. I had a rough draft a while ago but I like your version much better :)

File: src/v2/guide/conditional.md (hunk 1 above)
Comment (chrisvfritz) [inline src/v2/guide/conditional.md]: Good catch! I just added a function to escape HTML in sidebar links.

**Comment by LinusBorg** [comment]
Yeah this is really good! It shows that you are an educator - the way you introduce the upside of this behaviour before introducing a way to handle the "downside" is great!

And I think the message is transported very clearly.

**Pull request merged**